dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf

```diff
@@ -46,7 +46,19 @@ use=web, web=http://dynamic.zoneedit.com/checkip.html
 {%  if helpers.exists('OPNsense.DynDNS.accounts.account') %}
 {%    for account in helpers.toList('OPNsense.DynDNS.accounts.account') %}
 {%        if account.enabled|default('0') == '1' %}
-{%            if account.service == 'nsupdatev4' %}
+{%            if account.service == 'cloudflare' %}
+protocol=cloudflare
+zone={{account.zone}}
+{%            elif account.service == 'gandi' %}
+protocol=gandi
+zone={{account.zone}}
```

Comment:
While there isn't a TTL field for user input, may be we should add the following or it would be Gandi's default TTL which is 3h
`ttl={{OPNsense.DynDNS.general.daemon_delay|default('300')}}s`
(Taken from `daemon=` at line 1 of this file. Note that there is a trailing `s` because for gandi only, it uses T_DELAY type. Others use the T_NUMBER type.)